### ISSUE TITLE
Public Server Applications! [Apply In the Comments of This Issue!]

### ISSUE THREAD

**Issue by EPIICTHUNDERCAT** [Team-Rapture/Modpack-Hypothermia#23]
In Game Name:
Discord Name:

Age:

If Played on Server How long have you played?(check by going to the server info book ingame and go to leaderboards then time played):

Why should you be accepted?:

How long have you played modded?:

Where did you find Hypothermia?:

If a base is unclaimed what do you do?:

Provide more about yourself, help us understand who you are:

JOIN THIS DISCORD IT IS MANDATORY: https://discord.gg/aJe62Tq

**Comment by Unseenfury** [comment]
Unseenfury
15 year old
38.70*h
cuz i wanna help everyon and terraform the planet into grass instead of snow 🥉 

since the begining of minecraft, one of my friend presented t me minecraft when he had modded it

i got it from ThenonlyBentley

well i dont really know what to do, i just leave it here and dont claim it

well i'm a kind person who likes to help ppl out but when my nerv limits are passed i can be a bit controller in my team but i rarely do that, i live in canada and i speak 2 different language french canadaian and english as supose to now,okay goodbye ^-^

**Comment by NickPlaysNow** [comment]
IGN:ShadoMaster
Age:15
Time:43.53
Why: I want to play hypothermia with other people and help them survive in a desolate place.
Where: TheOnlyBentley
How Long: So long I can't remember
What to do: Leave it alone
More about myself: I like to play video games and I'm from America. Also I like to build large elaborate cool awesome buildings (so you should invite me to a build team to build a really cool spawn)

**Comment by c0lev** [comment]
IGN: colevelliott2000
Age: 16
Time: 13.36H
Why: I want to play on a server where I can have fun with others without getting griefed :)
Where: On the Curse Launcher
How Long: I don't even remember. More time than I can remember I guess. XD
What to do: I stay away from it and alert the person if I see them.
More about myself: I'm from America. I spend most of my free time playing modded minecraft. I have experience in most tech mods and quite a few magic mods as well. 

I would love to join the whitelist server, I hope you take me into consideration. Thanks!

**Comment by VeteranKyl** [comment]
In Game Name: Adventure_Drake

Age: 18

If Played on Server How long have you played?: New to the server.

Why should you be accepted?: I'm an avid modded Minecraft player that's looking for a private server to play on. I'm very experienced in modded Minecraft and glad to help out new and old players alike.

How long have you played modded?: Several Years (+3)

Where did you find Hypothermia?: Curse

If a base is unclaimed what do you do?: Leave it alone and contact the owner if there's a way to figure out who they are and get in touch.

Provide more about yourself, help us understand who you are: I'm a experienced minecraft player, both vanilla and modded. I live in the USA and spend my free time playing and doing programing on the side. I prefer playing in small groups vs large public servers, and I'm glad to help out new players with learning mods or getting started.

**Comment by Rymen** [comment]
In Game Name: x_Samm_x
Age: 25
If Played on Server How long have you played?(check by going to the server info book ingame and go to leaderboards then time played): cant remeber i think liek 10-20 hours maybe more

Why should you be accepted?: i tend to be helpful and non toxic

How long have you played modded?: since i started playing minecraft when it was 1.6.4

Where did you find Hypothermia?: curse

If a base is unclaimed what do you do?: just keep going

Provide more about yourself, help us understand who you are: im helpful and caring and fun to be aorund

**Comment by williamfriis** [comment]
1-In Game name williamfriis
2-Age 17
3- havnt played on the hypothermia server
4- well i like to be around people ingame and help people and do silly stuff so yeah and i am extream friendly
5- since 1.5.2 i started in tekkit
6- curse 
7- nothing just say to the owner of the base that please claim it for you own good so nobody is going to take it and just keep going
8-i am a friendly person who is   going to help is needed and going out to do random friendly stuff and still i am not toxic

**Comment by NoSkillPlays** [comment]
In Game Name: NqSkill_
Age: 14
If Played on Server How long have you played?(check by going to the server info book ingame and go to leaderboards then time played): 0

Why should you be accepted?: I am a good player, respectful, and good at modpacks.

How long have you played modded?: 9 months

Where did you find Hypothermia?: Friend told me and we are going to try to play. 

If a base is unclaimed what do you do?: Tell a staff.

Provide more about yourself, help us understand who you are: I am respectful and trustworthy.

**Comment by vertex101** [comment]
In Game Name: Vertex101
Age: 30
If Played on Server How long have you played?(check by going to the server info book ingame and go to leaderboards then time played): ummm idk i cant get on the public server to get the info

Why should you be accepted?: ummm i am a good person

How long have you played modded?: a couple years

Where did you find Hypothermia?: battery3996, emokiba

If a base is unclaimed what do you do?: run away from it and no touchie

Provide more about yourself, help us understand who you are: i am me and i hang out around alot of the streamer that deal with this pack if you wanna know more feel free to ask

JOIN THIS DISCORD IT IS MANDATORY: https://discord.gg/aJe62Tq

**Comment by LowerMercury** [comment]
1. My minecraft username is: Lower_Mercury
2. My age is 13
3. I can't go on the server right now but last I.p checked less than .3 hours away from having 30 so 29.7
4. I have never been one to grief steal and usaully am very respectful to my fellow neighbors 
5. Few years, I am usaully pretty good at figuring mods I don't know out
6. I found this server on the description of the hypothermia server which was on curse
7. Put a sign down in their base to let them know it's not claimed
8. I put a lot of time in mod packs and am very happy for the reset. I'm glad less active players that quit because of being to far ahead will be playing again

**Comment by gibbo100077** [comment]
In Game Name: gibbo100 or gibbo100077
Age: 50
If Played on Server How long have you played?(check by going to the server info book in game and go to leader boards then time played):  I died and kicked me out of the server
Why should you be accepted?:  I am learning how to use server.. Want to play more
How long have you played modded?:  2 months
Where did you find Hypothermia?:  Watching broadcasters playing it
If a base is unclaimed what do you do?:   I do not no.. I want to learn more
Provide more about yourself, help us understand who you are:   I would like to play on a server and off.. Want to learn more about how to play.. Talk to other players.. All my kids are all grown up and moved out and away.. Need something to do a night

**Comment by ShadowWisp** [comment]
In Game Name:ShadowWisper
Age:21
If Played on Server How long have you played? has not luck.

Why should you be accepted?: really wanted to play the adequate server, where's no griefers or idiots which ruins the game.

How long have you played modded?: over 2 months in total playtime (year, at least in time i've started the modded plays)

Where did you find Hypothermia?: itsLewdicolo, twitch streamer

If a base is unclaimed what do you do?: look who was it's owner, than live in there if there's noone plays it. If there's owner in there, leave a comment(sign or any other thing), htat it's not claimed.


Russian semi-adequate player, tired of a non-adequate players.  want to play in any adequate server.

**Comment by McGoofer1124** [comment]
In Game Name : McGoofer
Age:14
time on sever : 29 hours ish
why should you be accepted: decent person will play often and report bugs
How Long played modded : few months
found hypothermia: curse
if there is a unclaimed base: will ignore it and just assume some one forgot to claim it 
frends with Hellbat24 and Kothar5

**Comment by Deriol** [comment]
In Game Name: Deriol
Discord Name: Deriol

Age: 38

If Played on Server How long have you played?(check by going to the server info book ingame and go to leaderboards then time played):  0

Why should you be accepted?:  I am a friendly and helpful person.  I would hope that once I get myself established I would be able to help out with community things on the server if possible :D

How long have you played modded?:  When modded minecraft first became a thing, and you had to edit your Minecraft folders yourself.

Where did you find Hypothermia?:  Discord

If a base is unclaimed what do you do?:  If I am just starting out and its getting dark, I may hide there for the night then move on.  If I am already far into the game I would just leave the base be in hopes that someone was just unfortunate and forgot to claim it and they will be back soon. 

Provide more about yourself, help us understand who you are:  Part-time employed, Retail, so I may seem to be on a lot during my off hours.  Friendly, chatful, may have questions at the beginning since im new to hypothermia.  Two cats, Two dogs.

**Comment by LePinkie1** [comment]
n Game Name: LePinkie
Discord Name: Le Pinkie

Age: 18

Why should you be accepted?: because i wish to be kind not cancerous

How long have you played modded?: not long just started two days ago XD

Where did you find Hypothermia?: through emokiba

If a base is unclaimed what do you do?: id leave it alone its not mine and the guy who's building it probably would be pissed if i didn't

Provide more about yourself, help us understand who you are: i'm just your average team fortress 2 playing smuck but resonantly i'v been getting back into min craft so yeah

**Comment by ForeverSoloAlone** [comment]
In Game Name: ForeverSoloAlone
Discord Name: ForeverSoloAlone

Age:19

If Played on Server How long have you played?(check by going to the server info book ingame and go to leaderboards then time played): n/a

Why should you be accepted?: i love playing minecraft with others

How long have you played modded?: just got back in (i did play when FTB first came out)

Where did you find Hypothermia?: twitch mod packs

If a base is unclaimed what do you do?: Nothing (walk away)

Provide more about yourself, help us understand who you are: Less into building more in to making a powerfull base with everything in it

**Comment by peludo** [comment]
In Game Name: peludo71
Discord Name: peludo#5993

Age: 46

If Played on Server How long have you played?(check by going to the server info book ingame and go to leaderboards then time played):

Why should you be accepted?: I like to play MC with my daughter since 4yrs ago....

How long have you played modded?:  4yrs

Where did you find Hypothermia?: Curse

If a base is unclaimed what do you do?: First of all, I do not check that...I just play in my base and visit friends bases to hang up for a while or do common projects together.

Provide more about yourself, help us understand who you are:  I am a computer Science Engineer, work for a hosting company and former Oracle employee for 17yrs, and I like to play MC with my daughter......it will be nice to have her accepted too:  cowy102

**Comment by osmanosman88** [comment]
In Game Name: OsmanOsman
Discord Name: Osman_Best

Age: 17

If Played on Server How long have you played?(check by going to the server info book ingame and go to leaderboards then time played): 

Why should you be accepted?: I play a lot of modded MC almost done with a po2 run so looking for something to play on next.

How long have you played modded?: Since tekkit 1.2.5 (classic)

Where did you find Hypothermia?: Through Curse

If a base is unclaimed what do you do?: I usually try to find the original owner and tell a mod. 

Provide more about yourself, help us understand who you are: Obsessed with modded MC, I like creating packs, and play a lot of overwatch. 
EDIT: Also obsessed with anime

**Comment by jamiansa** [comment]
In Game Name: jamiansa
Discord Name: Jamiansa

Age: 15

If Played on Server How long have you played?(check by going to the server info book ingame and go to leaderboards then time played): 

Why should you be accepted?: i am wanting to start on this modpack and that me and my friend could play together

How long have you played modded?: 2 years

Where did you find Hypothermia?: curse launcher when going trought mods

If a base is unclaimed what do you do?: walk pass

Provide more about yourself, help us understand who you are: i am 15 years old boy from finland and i am wanting to help peoples too and try to help when i can and when i have stuff for helping

**Comment by farbythree** [comment]
In Game Name: masterzan
Discord Name: masterzan

Age: 18

If Played on Server How long have you played?(check by going to the server info book ingame and go to leaderboards then time played): New recruit

Why should you be accepted?: Ever since Agrarian Skies 1 came out I've been hooked on quest packs. I have a lot of experience with servers and modpacks after dropping hours into another modded server community (mineyourmind) however they have issues with porting in new modpacks past MC 1.7

This has lead me to search out new packs and communities to join, Hypothermia being one that hooked me instantly. It was a nice balance of tech, magic, difficulty, while at the same time incorporating new ways of acquiring resources that don't involve holes in the ground.

How long have you played modded?: This will be my 6th or 7th year. Back when I started up Tekkit was the only provider for easy to install packs. Ah yes, the good ol' days of overpowered EE. Took a few breaks every once and a while but some friends dragged me out of the Minecraft is lame phase and I've learned to love it with the same passion as when I was a kid.


Where did you find Hypothermia?: In my endless browsing of Curse (Twitch) modpack lists I stumbled upon Hypothermia.

If a base is unclaimed what do you do?: Leave a sign saying hello and point out their base is unclaimed. Maybe leave a small present like a sheep on a fence post cause why not. (Obviously not something that would hinder progress, or facilitate quicker progress in the pack)

Provide more about yourself, help us understand who you are: Just the average kid heading off to college with a free summer ahead of him. Seeing as I plan on majoring in Comp Sci I will already have a computer in front of me 😄

**Comment by orynstarheart** [comment]
In Game Name:Geiger_Rat
Discord Name: Oryn Starheart

Age:32

If Played on Server How long have you played?(check by going to the server info book ingame and go to leaderboards then time played): 300 hour

Why should you be accepted?: thousands of hours of minecraft and modded

How long have you played modded?: since Technic pack

Where did you find Hypothermia?: curse launcher

If a base is unclaimed what do you do?: leave it alone

Provide more about yourself, help us understand who you are:

**Comment by DiaDesaster** [comment]
IGN: DiaDesaster
Discord: DiaDesaster
Age: 19

Why should you be accepted?: Cause i want to play this new modpack with other nice guys and learn more about mods.

How long have you played modded?: I think 5-6 years i started with modded minecraft. 

Where did you find Hypothermia?: In curse (twitch) launcher.

If a base is unclaimed what do you do?: I talk to the guy if i know him that his base isnt claimed. 

Provide more about yourself, help us understand who you are: Im a 19 year old Gamer who live in Germany, i play videogames since i was 10 and i like to play with other people.

**Comment by Yeokrast** [comment]
IGN: Yeokrast
Discord: Yeokrast
Age: 33

Why should you be accepted?: I enjoy playing in a server environment with people who can become friends.  I am good with mods and interaction between them.

How long have you played modded?: I have been playing modded minecraft for 6ish years

Where did you find Hypothermia?: My friend Peludo mentioned it and then I looked it up. Looks great!

If a base is unclaimed what do you do?: If I knew who it was, I'd send them a pm letting them know that they may want to claim their base, otherwise I'd leave it alone.  I've been the victim of a partial base destroy because I hadn't fully claimed my base.

Provide more about yourself, help us understand who you are:  I am a minor gamer.  I used to be much more in my younger years before starting a family (I used to beta test all types of games for GameFly), but I am still strong to my roots.  I love Modded Minecraft and completing challenges/quests.  The more difficult or challenging the pack, usually the better -- unless it's just straight up ridiculous.  I've completed Invasion and it was fun, but am ready to move onto another challenge.

**Comment by rixyboy23** [comment]
In Game Name: rixyboy
Age: 23
If Played on Server How long have you played. About 25 hours on a single player world.
Why should you be accepted?: I like to help people out if I can.
How long have you played modded?: Since 1.2.5 
Where did you find Hypothermia?: On curse and Modii101.
If a base is unclaimed what do you do?: Keep running.
